homeassistant/loader.py

```diff
@@ -22,7 +22,7 @@
 from homeassistant.const import PLATFORM_FORMAT
 from homeassistant.util import OrderedSet
 
-# Typing imports
+# Typing imports that create a circular dependency
 # pylint: disable=using-constant-test,unused-import
 if False:
```

Comment:
Use `typing.TYPE_CHECKING`. See https://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles, and https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING.

Reply:
Done